Resources/Prototypes/_DV/Entities/Clothing/Eyes/hud.yml

```
@@ -58,3 +58,37 @@
   - type: Clothing
     sprite: Clothing/Eyes/Hud/syndagent.rsi
   - type: ShowSyndicateIcons
+
+- type: entity
+  parent: [ ClothingEyesBase, BaseToggleClothing ]
+  id: ClothingEyesPrescriptionDiagHud
+  name: prescription diagnostic hud
+  description: A poorly done and rushed mix between half of a pair of prescription glasses and a diagnostic HUD allowing you analyze the integrity and status of robotics and exosuits.
+  components:
+  - type: Sprite
+    sprite: _DV/Clothing/Eyes/Hud/prescdiaghud.rsi
+  - type: Clothing
+    sprite: _DV/Clothing/Eyes/Hud/prescdiaghud.rsi
+  - type: ToggleClothing
+    action: ActionToggleHudDiagnosticCamera
+    disableOnUnequip: true
+  - type: ItemToggle
+    onActivate: false # remove verb so you can't press E on it to add StationAiOverlay to a grid or something
+  - type: ComponentToggler
+    parent: true
+    components:
+    - type: StationAiOverlay
+      cosmetic: true
+  - type: Construction
+    graph: PrescriptionDiagHud
+    node: prescdiaghud
+  - type: Tag
+    tags:
+    - HudDiagnostic
+  - type: ShowHealthBars
+    damageContainers:
+    - Inorganic
+    - Silicon
+    - HumanoidSilicon
+  - type: VisionCorrection
+    correctionPower: 1.75 # The hud part decreases efficiency
```

It would make more sense to parent off of `ClothingEyesHudDiagnostic` instead. That would prevent us from having to (or forgetting to) make changes to the prescription version in the case that the regular HUD's component's change. (And it will inherit the components and abilities of the parent).

You will also want to add this tag to `ClothingEyesHudDiagnostic` as well. 

```yaml
  - type: Tag
    tags:
    - HudDiagnostic
```

```suggestion
  parent: ClothingEyesHudDiagnostic
  id: ClothingEyesPrescriptionDiagHud
  name: prescription diagnostic hud
  description: A poorly done and rushed mix between half of a pair of prescription glasses and a diagnostic HUD allowing you analyze the integrity and status of robotics and exosuits.
  components:
  - type: Sprite
    sprite: _DV/Clothing/Eyes/Hud/prescdiaghud.rsi
  - type: Clothing
    sprite: _DV/Clothing/Eyes/Hud/prescdiaghud.rsi
  - type: Construction
    graph: PrescriptionDiagHud
    node: prescdiaghud
  - type: VisionCorrection
    correctionPower: 1.75 # The hud part decreases efficiency
```